src/js/audio-player.js

```diff
@@ -42,6 +42,11 @@ export class AudioPlayer {
     player.current = 0;
   }
 
+  playOnLoop(key) {
+    const { player } = this.getAudioBy(key);
+    player.loop = true;
+  }
+
```

Comment:
This is a solid approach!

Looking at your implementation, you could technically combine this functionality into the `play` method itself.

For example:

```js
play(key, loop = false) {
    const { player } = this.getAudioBy(key);
    if (loop) player.loop = loop;
    player.play();
}
```

By default, it would play once, but if you pass `true` as the second parameter, it will enable looping.

If you feel this is a reasonable change, feel free to apply it. However, your current solution is equally valid, so it’s completely up to you!

--- 

This approach would make your code more readable and you don't have to call play on loop every time you play the background. Instead you can do this by providing a single argument.